### DISCUSSION TITLE
CI fails. How do I update github actions to Node.js 20?

### DISCUSSION THREAD

**Discussion by ocots** [control-toolbox/control-toolbox.github.io#54] [Q&A]
See [this](https://discourse.julialang.org/t/how-do-i-update-github-actions-to-node-js-20/110536). Check `dependabot.yml`.

[Update add Julia registry](https://github.com/julia-actions/add-julia-registry).

**Comment by amontoison** [comment]
You just need to update the following line:
```
julia-actions/setup-julia@v1
```
by
```
julia-actions/setup-julia@v2
```

**Reply by PierreMartinon** [reply]
Thanks @amontoison ! I use @ latest in CTDirect, are you aware of any drawbacks ?

**Reply by amontoison** [reply]
No, both are correct. Using `@latest` could be risky only for some complex GitHub Actions.
